config.mk

```diff
@@ -19,9 +19,11 @@ LDSCRIPT := ldscript
 NON_MATCHING ?= 0
 ifeq ($(PLATFORM),gba)
 ifeq ($(DEBUG),1)
+    PORTABLE     := 0
     NON_MATCHING := 1
     LDSCRIPT := $(LDSCRIPT)_modern.txt
```

Comment:
So why does debug use the modern ldscript?

Reply:
I was using that for testing, but you're right that it should be using the same for both.